Description: fix paths to FastCGI scripts
Origin: vendor
Forwarded: not-needed
Author: AGOSTINI Yves <agostini@univ-metz.fr>
Reviewed-by: Nicholas Bamber <nicholas@periapt.co.uk>,
 gregor herrmann <gregoa@debian.org>
Last-Update: 2019-07-17

--- a/lib/CGI/Fast.pm
+++ b/lib/CGI/Fast.pm
@@ -215,7 +215,7 @@
 FastCGI scripts must end in the extension .fcgi.  For each script you
 install, you must add something like the following to srm.conf:
 
-    FastCgiServer /usr/etc/httpd/fcgi-bin/file_upload.fcgi -processes 2
+    FastCgiServer /usr/lib/cgi-bin/file_upload.fcgi -processes 2
 
 This instructs Apache to launch two copies of file_upload.fcgi at
 startup time.
@@ -233,7 +233,7 @@
 webserver to connect to an external FastCGI server, you would add the following
 to your srm.conf:
 
-    FastCgiExternalServer /usr/etc/httpd/fcgi-bin/file_upload.fcgi -host sputnik:8888
+    FastCgiExternalServer /usr/lib/cgi-bin/file_upload.fcgi -host sputnik:8888
 
 Two environment variables affect how the C<CGI::Fast> object is created,
 allowing C<CGI::Fast> to be used as an external FastCGI server. (See C<FCGI>
